source/docs/hardware-components/kit-and-hardware-guide/tetrix.rst

```diff
@@ -1,6 +1,9 @@
 Tetrix
 ======
 
+.. attention::
+   Tetrix is generally not recommended for use, due to its restricted flexibility and long term reliability. We are retaining these pages as historical reference for teams that continue to have and use TETRIX parts.
```

Comment:
lol

Reply:
You know, with robits on the horizon, fair.